README.md

```diff
@@ -26,9 +26,8 @@ Navigate to the ES, IR, or SR band for further details on specific scripts and p
 * [LS](./LS)
 
 ## SystemReady Security Interface Extension:
-The SystemReady Security Interface Extension certifies that firmware meets the requirements specified by the Arm [Base Boot Security Requirements specification](https://developer.arm.com/documentation/den0107/latest) (BBSR). The Security Interface Extension is now integrated to SystemReady SR, ES and IR bands, but not the LS band.
-Further details on Security Interface Extension is here:
-* [SIE](./SIE)
+The SystemReady Security Interface Extension certifies that firmware meets the requirements specified by the Arm [Base Boot Security Requirements specification](https://developer.arm.com/documentation/den0107/latest) (BBSR). The Security Interface Extension ACS is now integrated to SystemReady SR, ES and IR bands, but not the LS band.

```

Comment:
Security Interface Extension ACS is now integrated to SystemReady SR, ES and IR ACS prebuilt images.